src/pydocstyle/conventions.py

```diff
@@ -0,0 +1,105 @@
+"""This module contains the convention definitions."""
+
+from typing import Set
+
+from typing_extensions import Literal
```

Comment:
This import can be removed now

Reply:
Removed. The tests did not crash because `importlib-metadata` pulled in `typing_extensions` as a dependency.